### DISCUSSION TITLE
How to remove @spell from strings?

### DISCUSSION THREAD

**Discussion by antosha417** [nvim-treesitter/nvim-treesitter#3603] [Q&A]
In [lua/highlights.scm](https://github.com/nvim-treesitter/nvim-treesitter/blob/master/queries/lua/highlights.scm#L189) we now have `(string) @string @spell`. 
How can one override it and remove spell checking inside of strings?
For me it is a bit too much, I want spell checks only inside of comments. 

I'm trying to add `(string) @string` inside `after/queries/lua/highlights.scm` but it does not seem to remove `@spell` from stings. It does have effect thought. If it instead enter `(string) @keyword` I see how highlights changes. But `@spell` persists for some reason.

**Comment by GustavoKatel** [comment]
Looks like there's an issue [here](https://github.com/neovim/neovim/issues/20174) to allow this kind of customization, I'm curious if there's any workaround already

**Reply by antosha417** [reply]
The only workaround I can think of is completely override queries for desired language with `:h set_query()`

**Reply by GustavoKatel** [reply]
that's what I was thinking, thanks. I guess I will wait for this PR then: https://github.com/neovim/neovim/pull/20178
